src/Constants/Version.php

```diff
@@ -12,5 +12,5 @@
      *
      * @var string
      */
-    const VERSION = '0.1.2';
+    const VERSION = '0.4.0';
```

Comment:
Whoops, I missed that when making https://github.com/codeclimate/php-test-reporter/pull/64 -- thanks for catching it